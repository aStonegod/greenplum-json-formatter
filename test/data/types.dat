{
    "id": 1,
    "i2": 2,
    "i4": -2147483647,
    "i8": 9223372036854775807,
    "t": "Some Text",
    "v": "Varchar data",
    "ts": "2013-07-02 16:09:07.838681",
    "f4": 4.4,
    "f8": 8.8
}
