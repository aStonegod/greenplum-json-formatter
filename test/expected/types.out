1|2|-2147483647|9223372036854775807|Some Text|Varchar data|2013-07-02 16:09:07.838681|4.4|8.8
